src/core/mantle/atoms/compute.ts

```diff
@@ -157,6 +142,45 @@ export function computeAtom(cache?: typeof globalDataFeaturesCache) {
     await set(compute, undefined);
   });
 
+  const writeComputedFeatures = atom(null, async (get, set, value: ComputedFeature[]) => {
```

Comment:
We can use `writeFeatures` atom, but delegated data is evaluated before invoking `writeFeatures`.
So I think we can skip the process, and we should use `writeComputedFeatures` to cache ComputedFeature directly.
Do you think about this? @rot1024